### ISSUE TITLE
Documentation: Location of config file for Linux

### ISSUE THREAD

**Issue by skipcress** [joncampbell123/dosbox-x#3590]
Hello,

On the wiki Welcome page, it says the config file location for Linux is:
~/.config/dosbox-x/dosbox-x-<version number>.conf

However, for the Flatpak, the config file is here:
~/.var/app/com.dosbox_x.DOSBox-X/config/dosbox-x/

Just a suggestion: can the Welcome page be updated to list the Linux Flatpak config file location, as well? Since the Flatpak auto updates, I imagine a lot of people like me have to update their config files quite regularly, and it would be helpful to have that location better documented.

Thanks!
Kirk

**Comment by rderooy** [comment]
This is actually documented on the flathub git repo readme.
https://github.com/flathub/com.dosbox_x.DOSBox-X

**Comment by skipcress** [comment]
Yes, I understand that, that’s where I ultimately found it; however, I still feel that there would be some benefit to adding it to the main wiki Welcome page, since that’s where a person is likely to check first. I personally found it a bit frustrating that it wasn’t there when I first was trying to find it. It’s also arguably a little misleading to say that the config file path for “Linux” is any one of the two paths, since they’re both applicable to Linux. I’d say that’s especially true given on https://dosbox-x/ (the main page) under “Linux version:” it just gives a link to the Flatpak, there are no instructions for compiling it locally and installing it that way.
